console/src/telemetry/Actions.ts

```diff
@@ -12,7 +13,15 @@ const SET_CONSOLE_OPTS = 'Telemetry/SET_CONSOLE_OPTS';
 const SET_NOTIFICATION_SHOWN = 'Telemetry/SET_NOTIFICATION_SHOWN';
 const SET_HASURA_UUID = 'Telemetry/SET_HASURA_UUID';
 
-const setConsoleOptsInDB = (opts, successCb, errorCb) => (
+type Opts = {
+  telemetryNotificationShown: boolean;
+  disablePreReleaseUpdateNotifications: boolean;
+};
+type SuccessCb = (arg: object) => void;
+type ErrorCb = (arg: Error) => void;
+type SetConsoleOptsInDB = (opts: Partial<Opts>, successCb: SuccessCb, errorCb: ErrorCb) => (dispatch: Dispatch<any>, getState: any) => void
+
+const setConsoleOptsInDB: SetConsoleOptsInDB = (opts, successCb, errorCb) => (
```

Comment:
Let's remove above types (as they are not needed anywhere else, and just inline types of `setConsoleOptsInDB`:
```ts
const setConsoleOptsInDB = (
  opts: Opts,
  successCb: (arg: object) => void,
  errorCb: (arg: Error) => void
) => (
  dispatch: Dispatch<any>,
  getState: () => { telemetry: TelemetryState }
) => {
```